### DISCUSSION TITLE
Privacidade com Privoxy e ZeroHosts

### DISCUSSION THREAD

**Discussion by RatoX** [frontendbr/forum#366] [Perguntas e Respostas]
Então tem um bom tempo que eu uso o velho combo:

[DNSCrypto](https://dnscrypt.org/) + [Privoxy](https://www.privoxy.org/) + [ZeroHosts File](https://github.com/StevenBlack/hosts) + [VPN](https://vpn.ccrypto.org/)

Além dos de sempre: [KeePassX](https://www.keepassx.org/) e um email de back no com hidden services.

O que vocês usam pra navegar com segurança e privacidade na rede de computadores?

**Comment by willycamargo** [comment]
🤔

Eu acho interessante o assunto, mas eu particularmente simplesmente ligo o wifi, abro o chrome e começo a navegar. Não vejo necessidade de fazer mais do que isso (pelo menos pra mim que não trabalho com nenhum tipo de dado considerado cauteloso).

Mas vou acompanhar a issue para entender melhor sobre o assunto =)

**Comment by thulioph** [comment]
@RatoX Tem alguma razão pra se ter todo esse ambiente configurado? Ultimamente tenho lido bastante sobre a deep web e pra quem acessa, requer um cuidado extra.

Pergunto porque me interesso sobre o assunto e queria trocar figurinha com quem entende :D

**Comment by karlprieb** [comment]
Alguém ai usa a ZeroNet?

**Comment by RatoX** [comment]
@thulioph Basicamente, minha privacidade. Eu sou militante de alguns movimentos sociais, também, por isso preciso de uma camada extra de segurança. 

@karlprieb  Não conhecia a ZeroNet, pelo que eu entendi ela funciona sobre camadas que nem o Onion Protocol porém baseada em blockchain?   

@willycamargo Cara, você sempre tem que cuidar de sua privacidade afinal ela é sua. Você não precisa estar fazendo nada de errado para querer se proteger de vigilância constante. Imagine um BBB onde você é a peça central, essa é a nossa vida atual depois de Snodew.

**Comment by RatoX** [comment]
Ah nada do que eu citei inicialmente te garante anonimidade na internet.

**Comment by karlprieb** [comment]
@RatoX Não, ele não usa blockchain, só a criptografia do bitcoin para os hashes e autenticações. O resto é P2P normal, porém tu pode te conectar a partir do Tor para aumentar um pouco a tua anonimidade.
